### PR TITLE
add signup and login

### PR DESCRIPTION
**Pull request by jj49411**

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<%= form_for(:session, url: login_path) do |f| %>
```
Comment (Hyan18) [inline app/views/sessions/new.html.erb:2]: form_for is soft-deprecated so should use form_with instead

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    @user = User.find_by_email(params[:session][:email])
+
```
Comment (Hyan18) [inline app/controllers/sessions_controller.rb:7]: Run rubocop

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+
+  has_secure_password
+
+  validates_presence_of :name
+  validates_presence_of :email
+  validates_uniqueness_of :email
+  validates_presence_of :password
+  # validates_confirmation_of :password
```
Comment (Hyan18) [inline app/models/user.rb:9]: Is this still needed for something later?

**Review by Hyan18 — CHANGES_REQUESTED** [review]
Overall seems fine. Good job guys!

Just remember to run Rubocop regularly (run it before merging)

File: app/models/user.rb (hunk 3 above)
Comment (jj49411) [reply app/models/user.rb:9]: #validates_confirmation_of :password 
this line you mean?

File: app/controllers/sessions_controller.rb (hunk 2 above)
Comment (jj49411) [reply app/controllers/sessions_controller.rb:7]: I pulled it on the master but not this branch so can't run rubocop

Comment (DawidSzpener) [reply app/controllers/sessions_controller.rb:7]: rubocop has 72% coverage

**Comment by DawidSzpener** [comment]
Lets talk about this pull request tommorow morning

File: app/models/user.rb (hunk 3 above)
Comment (Hyan18) [reply app/models/user.rb:9]: Yeah because it currently is just commented out

File: app/controllers/sessions_controller.rb (hunk 2 above)
Comment (Hyan18) [reply app/controllers/sessions_controller.rb:7]: > 
> 
> I pulled it on the master but not this branch so can't run rubocop

Ok nvm, we can do it after we merge then

**Pull request merged**